app/controllers/events_controller.rb

```diff
@@ -0,0 +1,50 @@
+class EventsController < ApplicationController
+  before_action :authenticate_user, only: %i[attend new]
+
+  def index
+    @past_events = Event.previous_events
+    @upcoming_events = Event.upcoming_events
+  end
+
+  def new
+    @event = Event.new
+  end
+
+  def create
+    user = User.find(session[:current_user_id])
+    @event = user.created_events.build(event_params)
+    pp @event
+    if @event.save
+      redirect_to event_path(@event), notice: 'Event Created successfully!'
+    else
+      render :new
+    end
```

Comment:
You are not displaying the errors to the user. When I try to create an invalid event I just can't, but I don't know why. Please improve that.